app.py

```diff
@@ -1,48 +1,42 @@
-import collections
 import datetime
 import logging
-import math
-import requests
 import time
 
-import numpy.random
+import numpy
 import pandas
-import plotly.express
+import plotly
```

Comment:
Why not import `plotly.express`? I'd like to see where the objects we use belong.